.github/workflows/test.yml

```diff
@@ -0,0 +1,35 @@
+name: Run Test
+on: [push]
+
+jobs:
+  build:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [10.x, 12.x, 14.x]
+        mongodb-version: [4.0, 4.2, 4.4]
```

Comment:
The lowest MongoDB version currently supported by Agenda is 3.4. Could you please add 3.4 and 3.6 to the list if possible?

https://github.com/agenda/agenda/blob/master/.travis.yml#L14

Reply:
Sure!